packages/keystone/prepareKeystoneApp.js

```
@@ -4,16 +4,31 @@
+    logger.info({msg: 'start prepare keystoneExpressApp', data: { entryPoint, excludeApps, processCwd: process.cwd() }})
     const dev = process.env.NODE_ENV === 'development'
+
     const {
         keystone,
         apps,
         configureExpress,
         cors,
         pinoOptions,
     } = (typeof entryPoint === 'string') ? require(entryPoint) : entryPoint
+
+    logger.info({msg: 'require module', data: {
```

ESLint

[error] A space is required after '{'.

(@typescript-eslint/object-curly-spacing)